packages/hub/services/relay.ts

```diff
@@ -82,6 +82,33 @@ export default class RelayService {
     let body = await response.json();
     return body.txHash;
   }
+
+  async registerProfile(userAddress: string, did: string): Promise<string> {
+    let relayUrl = getConstantByNetwork('relayServiceURL', web3config.layer2Network);
+    if (!provisionerSecret) {
```

Comment:
Should we use the "provisioner secret", which is used for prepaid card provisioning, for registering merchants as well? Or would it be better to add another secret to the relayer,  the "merchant registrar secret"? This would be clearer to identify which secret is for what action. But this doesn't really scale well in case we add even more actions to the relayer... Or are we okay with "provisioner secret" being a good umbrella name for the trusted actions we run thru the relayer? Also pinging @habdelra for this one

Reply:
I can’t think of a reason to have separate secrets unless it feels safer to isolate a leak to one contract. But this is definitely wild speculation for me hehe

Reply:
i agree. we can segment them in the future, but I think the funds at risk right now is low enough to not worry about having separate secrets

Reply:
Thanks, noted!